yogstation/code/modules/admin/verbs/fix_air.dm

```diff
@@ -18,3 +18,41 @@
 			GM.parse_gas_string(F.initial_gas_mix)
 			F.copy_air(GM)
 			F.update_visuals()
+
+/client/proc/fix_air_z()
+	set name = "Fix Air on current Z-level"
+	set category = "Misc.Unused"
+	set desc = "Fixes air on the entire z-level, temporarily disables atmos"
+
+	if(!holder)
+		to_chat(src, "Only administrators may use this command.", confidential=TRUE)
+		return
+	if(!check_rights(R_ADMIN,1))
+		return
+
+	if(alert("Do you want to reset air on the entire z level?", "Fix Air- Z level", "No", "Yes") != "Yes")
```

Comment:
Maybe you could add the z-level they're on to the prompt for those special admins :)

Reply:
No, its called natural selection if you get removed for thinking youre fixing air on CC when youre clearly on the station

Reply:
I agree with alexkar